### ISSUE TITLE
[roads] end of oneway unclear

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4911]
**[Submitted to the original trac issue database at 8.33pm, Monday, 22nd July 2013]**

An [http://www.openstreetmap.org/?lat=50.620504&lon=5.660834&zoom=18&layers=M inhabitant of this place] draws my attention (aren't some people nice?) on the oneway street Jean Leclercq. He says that the map makes it unclear that the oneway restriction stops half way, at the crossing with street Jean Jaurs. He says that there should be an arrow there to show that end.[[BR]]
I'm taking the idea one step further. An additional arrow is a hint and not a certainty. Could it be easily possible, when a oneway stops in the middle of a street, to draw a clear double headed arrow at the beginning of the unrestricted segment?[[BR]]
This is similar to Ticket #4338, but different. They should be considered in parallel. Thanks.

**Comment by tomhughes** [comment]
**Author: Papou**
**[Added to the original trac issue at 1.43am, Thursday, 29th August 2013]**

This situation is similar to the top-to-tail arrows traffic signs warning someone leaving a one-way street that he enters a two-way street and that he can face traffic from there on.  I tend to think that the arrows (single and double headed) must be placed at the ends of all the streets that meet at a crossing when one them is one-way in addition to spread on a street based on its segments alone.[[BR]]
[http://www.openstreetmap.org/?mlat=50.50715&mlon=5.66126#map=19/50.50715/5.66126 Also, consider this roundabout.] Streets where you can exit have an arrow or none and the street where you cannot exit is among those without an arrow.

**Comment by tomhughes** [comment]
**Author: math1985**
**[Added to the original trac issue at 9.09pm, Monday, 2nd June 2014]**

I think this would cause too much clutter. We also don't really have a way to implement this. I will therefore close this as wontfix.
